apps/src/lab2/views/components/instructions.module.scss

```diff
@@ -145,16 +145,15 @@
   border: dashed $neutral_white 2px;
 }
 
-.buttonNext {
-  font-size: 14px;
-  background-color: $brand_secondary_default;
-  border-color: $brand_secondary_default;
-  color: $neutral_light;
-  width: 100%;
-  padding: 5px 10px;
-  border-radius: 4px;
-  height: 38px;
-  margin: 0 0 10px 0;
+.button {
+  width: 100%;  
+  margin-bottom: 10px !important;
+  &:disabled, &[aria-disabled="true"] {
```

Comment:
Curious, do the default DSCO button styles not work here?

Reply:
Thanks for catching this! I added this back when I was using `<button>` and added this before deciding to use the DSCO button. Will remove.

Reply:
FYI - After discussing with @breville , we're going to keep the style as is to not making any changes, even if subtle, before the music lab launch. After the two launches, we can make a lab2-wide decision on what buttons to use for common components. Thanks!